spell/mom-onboarding-checklist.md

```diff
@@ -0,0 +1,30 @@
+# Mom Onboarding Checklist
+  * [ ] Deployed Contracts
+    * [ ] `Mom`
+      * [ ] deployed via deployer
+        * [ ] ensure different deployer is used for mainnet and testnet to avoid contracts with same address and different sources
+        * [ ] contract is verified on etherscan
+          * [ ] ensure solc version matches source
+          * [ ] ensure optimization matches source
```

Comment:
what is `source` here? we don't usually specify optimizations in the repo iirc

Reply:
yeah, this could be rephrased better, it refers to repo optimization configs

Reply:
I've specified, as done in other checklists (e.g. RWA), that optimization should be off considering the compact nature of the mom contracts, done in 104b4db